### DISCUSSION TITLE
Chinese translation

### DISCUSSION THREAD

**Discussion by greenhsu123** [Green-Software-Foundation/learn#171] [General]
Community members have proposed that they would like to translate the V1 into Chinese. 

from @Jjing-Liang:
_I'm Jing, a developer from China, I learned through GSF newletter that there is an Italian version, and some of us want to translate a Chinese version so that more Chinese people can learn about green software.
Can we refer to the previous process to participate in it? Or what is the process or requirements for translation participation?_

For details on the previous process, please look at the Italian [MR](https://github.com/Green-Software-Foundation/learn/pull/151). 

Tl:dr: once you and your team have finished translating the principles into Chinese, please raise a PR. We would then start a translation review there while connecting you with the GSF's designer to work on the graphics translation. 

cc: @jawache, @Oleg-Zhymolokhov

**Comment by Jjing-Liang** [comment]
Thanks @greenhsu123.

**Comment by russelltrow** [comment]
Hello @Jjing-Liang, do you still wish to contribute a Chinese translation of the course?

Russ

**Reply by Jjing-Liang** [reply]
Thank you for asking about the progress. 
Yes! My coworkers and I have been working on the translation lately and will be raising a PR later!

**Reply by russelltrow** [reply]
Hi @Jjing-Liang that is fantastic to hear. We look forward to it.

**Reply by Jjing-Liang** [reply]
Hi @russelltrow 
Just wanted to shoot you a quick update on the translation we've been working on. We're currently in the final review stage, and things are looking great!

The good news is, we're on track to wrap up the translation by the end of November 2023. Once we're done with the final review, we'll send a PR with the translated content. That way, you can take a look, give us your feedback, and make any necessary tweaks.

We just wanted to keep you in the loop so that you know where we're at. We appreciate your patience and support throughout this process.

**Reply by russelltrow** [reply]
Hi @Jjing-Liang thank you so much for the update. We are really excited to receive the translation.

**Reply by Jjing-Liang** [reply]
Hi @russelltrow , we have created a [pull request](https://github.com/Green-Software-Foundation/learn/pull/207), please help review the content.
Thanks for the patience during this time!

**Comment by russelltrow** [comment]
Translation is live!
